backend/src/junior/dto/edit.dto.ts

```diff
@@ -1,4 +1,4 @@
-import { IsNotEmpty, IsPhoneNumber } from 'class-validator';
+import { IsNotEmpty, IsPhoneNumber, Length, IsPositive } from 'class-validator';
```

Comment:
IsPositive is defined but not used, assuming this should be applied to age (and maybe Finnish post code?)